### PR TITLE
[CHI-375] Drawer JavaScript documentation updated, added methods

### PR DESCRIPTION
**Pull request by janibolkvadze**
@jllr @mattnickles @dani-cl-madrid 

https://ctl.atlassian.net/browse/CHI-375

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/261/ for more details.

File: src/website/views/javascript/drawer.pug [hunk 1]
```diff
@@ -144,7 +144,31 @@ section.a-table.a-table__options.-bordered.-my--3
...
+    | returns 
+    b type void
+  h4 toggle() =&gt; void
+  p.-ml--2 Triggeres drawer 
+    b show/hide  events
+    br
+    | returns 
+    b type void
+
+h3 Preventing memory leaks
```
Comment (dani-cl-madrid) [inline src/website/views/javascript/drawer.pug]: **Preventing memory leaks** is h4 in all documentation pages but here. Please use it consistently. @jllr, do you think we should change all Preventing memory leaks to h3, or keep them as h4?

File: src/website/views/javascript/drawer.pug [hunk 2]
```diff
@@ -144,7 +144,31 @@ section.a-table.a-table__options.-bordered.-my--3
               | Lets the developer referer the target programmatically instead of by data-target attribute.
               | It accepts Element object and xpath selector string.
 
-h4 Preventing memory leaks
+h3.mt--2 Methods
+.p.-text.-mb--3
+  .p-text There are three
```
Comment (dani-cl-madrid) [inline src/website/views/javascript/drawer.pug]: Please use markup used in https://github.com/dani-cl-madrid/ux-chi/blob/ab292cf61329270f2472cd1862603931b53eb0ca/src/website/layouts/partials/stencilDoc.pug

File: src/website/views/javascript/drawer.pug (hunk 1 above)
Comment (jllr) [reply src/website/views/javascript/drawer.pug]: @dani-cl-madrid Keep them as h4

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/278/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/280/ for more details.

**Pull request merged**